flaskapp/resources/heretraffic.py

```diff
@@ -0,0 +1,25 @@
+import requests
```

Comment:
6 spaces indentation instead of standard 4

Reply:
So I discovered my IDE will literally hide away this indentation issue! It was 4 spaces in my IDE, but in all my other text editors it was 6 lol. Fixed.